lib/src/polygon_to_line.dart

```diff
@@ -0,0 +1,69 @@
+import '../helpers.dart';
+
+/// Converts a [Polygon] to [LineString] or [MultiLineString] or a [MultiPolygon] to a
+/// [FeatureCollection] of [LineString] or [MultiLineString].
+/// Returns [FeatureCollection] or [Feature<LineString>] or [Feature<MultiLinestring>]
+/// example:
+/// ```dart
+/// var poly = Polygon(coordinates:
+///  [
+///   [
+///   Position.of([125, -30]),
+///   Position.of([145, -30]),
+///   Position.of([145, -20]),
+///   Position.of([125, -20]),
+///   Position.of([125, -30])
+///   ]
+/// ]);
+/// var line = polygonToLine(poly);
+/// //addToMap
+/// var addToMap = [line];
+/// ```
+polygonToLine(GeoJSONObject poly, {Map<String, dynamic>? properties}) {
+  var geom = poly is Feature ? poly.geometry : poly;
+
+  (poly is Feature)
+      ? properties = poly.properties
+      : properties = <String, dynamic>{};
```

Comment:
either use the `? :` expression properly like this:
```dart
properties = poly is Feature ? poly.properties : {};
```
or use an if-else statement